opendebates/templates/opendebates/snippets/idea.html

```diff
@@ -87,12 +87,17 @@
         {% endcomment %}
       </div>
 
+      <div class="flags">
+        <a href="{% url 'report' idea.pk %}">{% trans "Report" %}</a>
+        <a href="{% url 'merge' idea.pk %}">{% trans "Merge" %}</a>
```

Comment:
Let's not show the `Merge` link on questions that have already been merged into another.